packages/sdk/src/index.ts

```diff
@@ -6,15 +6,23 @@ export * as across from "./across";
 export { default as Coingecko } from "./coingecko";
 export { default as Multicall } from "./multicall";
 export { default as Multicall2 } from "./multicall2";
+export { default as BlockFinder } from "./blockFinder";
 
 // types
 import type { TypedEventFilterEthers as TypedEventFilter, TypedEventEthers as TypedEvent } from "@uma/contracts-node";
-import { Contract, ethers, Signer, providers, Event } from "ethers";
+import { Contract, ethers, providers, Event } from "ethers";
 import { Provider } from "@ethersproject/providers";
+import { Signer } from "@ethersproject/abstract-signer";
```

Comment:
did this helped to get rid of the build error? I had to do the same thing with the `Provider` type. `providers.Provider` conflicted with the Provider used by the contracts & factories generated by the typechain

Reply:
yea... i think the issue is that `@ethersproject` packages seem to be different versions, even if u have the same `ethers` defined at times.  I think the real fix was this https://github.com/across-protocol/across-frontend/pull/74. But this is a lot of guesswork from me, and  i literally kept trying tihngs till it worked. we will see what CI says